go.mod

```diff
@@ -32,7 +32,6 @@ require (
 	golang.org/x/net v0.0.0-20220812174116-3211cb980234
 	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
 	google.golang.org/grpc v1.47.0
-	inet.af/netaddr v0.0.0-20220617031823-097006376321
```

Comment:
@schrej Doesn't matter at all. But are you also confused why we didn't have to bump `go 1.17` in the go.mod file to use `netip.ParseAddr`? :)

Reply:
Interesting. The compiler does heavily complain about generics if it doesn't say `1.18` in `go.mod`, but it doesn't seem to care about packages that aren't available in an earlier release.

It doesn't compile with 1.17, so technically the project needs to change to `1.18` in `go.mod`, or revert this change. Probably doesn't have any impact to consumers of APIs though, as its unlikely that someone imports the webhooks :smile:

Reply:
There is an issue for it: https://github.com/golang/go/issues/46136

Reply:
Thx for looking that up!

I think for CAPI main it's fine. We're probably days away from bumping to Go 1.19